init.sh

```diff
@@ -0,0 +1,7 @@
+if [[ $(cargo --version) ]]; then
+    echo "Found cargo"
+else
+    curl https://sh.rustup.rs -sSf | sh -s -- -y
+    source $HOME/.cargo/env
```

Comment:
Same, if the `env` script fails, `init.sh` should abort:
```suggestion
    source $HOME/.cargo/env || exit
```